src/docker-osx-dev

```diff
@@ -1031,6 +1033,23 @@ function install {
 }
 
 #
+# Fetch the latest docker-osx-dev script, and then run install
+#
+function update_self {
+  log_info "Updating to the latest docker-osx-dev"
+  current_file=$(which docker-osx-dev)
+  log_info "  -- Backing up $current_file to $current_file.old"
+  cp $current_file "$current_file.old"
+
+  latest_release="https://raw.githubusercontent.com/brikis98/docker-osx-dev/master/src/docker-osx-dev"
```

Comment:
Store this as a `readonly` variable, all in caps, at the top of the file, where the rest of the constants are defined